typeguard/node.ts

```diff
@@ -357,7 +357,7 @@ export function isJsxExpression(node: ts.Node): node is ts.JsxExpression {
     return node.kind === ts.SyntaxKind.JsxExpression;
 }
 
-export function isJsxFramgment(node: ts.Node): node is ts.JsxFragment {
+export function isJsxFragment(node: ts.Node): node is ts.JsxFragment {
```

Comment:
I'd rather keep the misnamed function around to avoid a breaking change. It can just defer to the new function and is marked as `@deprecated` in jsdoc.

Reply:
That is a really good idea. I will make that change.